tests/__sf_section_kubecontext.test.fish

```diff
@@ -11,7 +11,7 @@ test "Prints section"
 		echo -n "at "
```

Comment:
Could we please add the following test cases:
- `SPACESHIP_KUBECONTEXT_NAMESPACE_SHOW = "false"` and kubecontext isn't shown
- `$kube_context = "default"` and kubecontext isn't shown

Reply:
Great idea. Did it!